src/System.Buffers.Experimental/System/Range.cs

```diff
@@ -1,26 +1,78 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace System {
-    public struct Range
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System
+{
+
+    public struct Range : IEnumerable<int>
```

Comment:
Should we call it Range32?

Reply:
Sounds like Win32. It's better to use `IntRange` or just `Range`.